src/main/resources/l10n/JabRef_fa.properties

```diff
@@ -1149,7 +1149,7 @@ Settings=
 
 Shortcut=
 
-Show/edit_BibTeX_source=
+Show/edit_%0_source=Montrer/�diter_le_source_BibTeX
```

Comment:
@lenhard Can you change the encoding of the farsi file? Should be no problem there are no translations in this file.